examples/asr/asr_webapp/docker_container_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Isn't it easier to put the building and running docker commands in the docs directly? Both ways are actually OK.

Reply:
In docs means it cant be executed. Rather keep it a runnable bash script.